wafer/schedule/templates/wafer.schedule/penta_schedule.xml

```diff
@@ -105,8 +105,7 @@
                       {% else %}
                         <description/>
                       {% endif %}
-                      <released>True</released>
-                      <license>{{ WAFER_VIDEO_LICENSE }}</license>
```

Comment:
Why is the license tag being removed?

Reply:
It does go hand in hand with being released